debian/changelog

```diff
@@ -1,3 +1,25 @@
+snapcraft (2.8.9) xenial; urgency=medium
+
+  [ Leo Arias ]
+  * autopkgtests: run the install examples tests in classic. (#481)
+    (LP: #1572764)
+
+  [ Matteo Bertini ]
+  * Fix typo in description: python2 -> python3. (#504)
+
+  [ Jamie Bennett ]
+  * Documentation: Use plugs instead of caps. (#507)
```

Comment:
I'm not sure this (or the one directly above) are particularly useful in the changelog.